lib/liquid/block_body.rb

```diff
@@ -1,4 +1,6 @@
 module Liquid
+  using RubyBackports
```

Comment:
Ruby 2.1 and 2.2 docs say: "You may only activate refinements at top-level, not inside any class, module or method scope." 2.3 says "You may activate refinements at top-level, and inside classes and modules."

May as well put this line at the top?

Reply:
Sure.  I guess it just wasn't scoped when I tested on ruby 2.1, but I also don't need it to be scoped.

Reply:
Oh right, it does make it awkward to make the Liquid::RubyBackport a private constant

Reply:
Well, it does pass on CI, not sure if the docs are just being defensive or if there's something else at play.

Reply:
Did some testing and found that the following works on ruby 2.1 but not in ruby 2.3 because ruby 2.3 scopes the refinement

```ruby
module Liquid
  using RubyBackports
end

module Liquid
  class BlockBody
```

I think the nested refinement is cleaner than having `using Liquid.const_get(:RubyBackports)` at the top-level or making Liquid::RubyBackports public.

Reply:
👍 so you can activate a refinement in a module, but it will be still globally scoped.